pyproject.toml

```diff
@@ -1,31 +1,24 @@
 [build-system]
-requires = [ "setuptools", "setuptools_scm" ]
+requires = ["setuptools>=45", "setuptools-scm[toml]>=6.2"]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "ts-dream"
 description = "DREAM camera CSC."
 license = { text = "GPL" }
 classifiers = [ "Programming Language :: Python :: 3" ]
-urls = { documentation = "https://ts-dream.lsst.io", repository = "https://github.com/lsst-ts/ts_dream" }
-dynamic = [ "version" ]
-dependencies = [
```

Comment:
I would like to keep at least dependency (probably a new one) on ts_tcpip, like this:

```iff --git a/pyproject.toml b/pyproject.toml
index 3cf3fb5..0a465f1 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -9,6 +9,9 @@ license = { text = "GPL" }
 classifiers = [ "Programming Language :: Python :: 3" ]
 urls = { documentation = "https://ts-dream.lsst.io", source_code = "https://github.com/lsst-ts/ts_dream" }
 dynamic = ["version"]
+dependencies = [
+  "ts_tcpip",
+]
 
 [tool.setuptools.dynamic]
 version = { attr = "setuptools_scm.get_version" }
```